### PR TITLE
MPOC-287 - BUG Fix

### PR DESCRIPTION
**Pull request by simaomazive**
The system automatically clear the HF code already set previously when any dropdown field is selected.

### CODE REVIEW

File: ui/app/common/attributeTypes/directives/attributeTypes.js [hunk 1]
```diff
@@ -56,7 +56,10 @@ angular.module('bahmni.common.attributeTypes', []).directive('attributeTypes', [
                     return answer.conceptId === attributeValueConceptType.conceptUuid;
                 });
                 attributeValueConceptType.value = concept && concept.fullySpecifiedName;
-                $rootScope.typeOfRegistrationSelected = attributeValueConceptType.value;
+                if (attributeValueConceptType.value === 'NEW_PATIENT' || attributeValueConceptType.value === 'TRANSFERRED_PATIENT' || attributeValueConceptType.value === undefined) {
```
Comment (nthfloor) [inline ui/app/common/attributeTypes/directives/attributeTypes.js:59]: @simaomazive when I test this bug, I am unable to replicate locally. Without this change the FacilityCode is not affected when I select other drop downs on form.

is there a way for me to better replicate the issue?

File: ui/app/common/attributeTypes/directives/attributeTypes.js [hunk 2]
```diff
@@ -56,7 +56,10 @@ angular.module('bahmni.common.attributeTypes', []).directive('attributeTypes', [
...
+                    console.log(attributeValueConceptType.value);
```
Comment (amosh101) [inline ui/app/common/attributeTypes/directives/attributeTypes.js]: Just need to clear this console log.

**Review by amosh101 — COMMENTED** [review]
Clear Console log.

File: ui/app/common/attributeTypes/directives/attributeTypes.js (hunk 2 above)
Comment (simaomazive) [reply ui/app/common/attributeTypes/directives/attributeTypes.js]: @amosh101 done

**Review by amosh101 — APPROVED** [review]

**Pull request merged**